src/framework/mpe/qml/MuseScore/Mpe/ArticulationPatternControlsTab.qml

```diff
@@ -19,56 +19,62 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-
 import QtQuick 2.15
-import QtQuick.Controls 1.5
+import QtQuick.Layouts 2.15
```

Comment:
You used the wrong version here. Should be 1.15

Reply:
Thanks, fixed!